### ISSUE TITLE
ccgan

### ISSUE THREAD

**Issue by kareem337** [eriklindernoren/PyTorch-GAN#167]
If i want to add a layer which uses softmax for classification of multi-classes what line of code should i update? Thank you in advance.

**Comment by kareem337** [comment]
Yeah thank you but can you help me? I need to make the discriminator classify every image with its label not just see if it is real or fake (generated).
